Toolbox/scripts/dump_hosts.sh

```diff
@@ -5,7 +5,7 @@ TOPIC=Hosts
 DESCRIPTION="This script will dump the hosts file."
 
 #Volumes/files
-VOLUME=/fs/sda0
+. /eso/bin/PhoneCustomer/default/util_mountsd.sh
```

Comment:
On line 17 there also is this command? Doubled?

Reply:
I'll check, thanks